main.go

```diff
@@ -7,17 +7,23 @@ import (
 	"log"
 	"os"
 	"strings"
+
+	"github.com/status-im/go-maven-resolver/fetcher"
+	"github.com/status-im/go-maven-resolver/finder"
+	"github.com/status-im/go-maven-resolver/pom"
 )
 
 var l *log.Logger
 
-var workersNum int
-var requestTimeout int
-var reposFile string
-var ignoreScopes string
-var recursive bool
+var (
+	workersNum int
+	requestTimeout int
+	reposFile string
+	ignoreScopes string
+	recursive bool
+)
```

Comment:
Just idiomatic, it's also a lot of personal taste too, I don't really care too much about this sort of thing but the go community does